arcgis-java-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/UtilityNetworkTraceTool.java

```diff
@@ -72,8 +75,6 @@
 import javafx.scene.input.MouseEvent;
 import javafx.scene.paint.Color;
 
-import static java.util.stream.Collectors.groupingBy;
```

Comment:
Somewhere there should be a comment explaining how you can use `groupingBy` but avoiding the static import.

Reply:
https://github.com/Esri/arcgis-runtime-toolkit-java/pull/57#discussion_r950259610

Reply:
Thanks for explaining that, I've refactored